LoginViewController.swift

```diff
@@ -136,7 +138,7 @@ class LoginViewController: UIViewController, UITextFieldDelegate
     }
 
 //MARK: IBAction - Signin validation, storage, and segue
-    @IBAction func signInButtonTapped(_ sender: UIButton)
```

Comment:
sender needs to be optional because i'm calling it without a sender during textFieldShouldReturn